tests/queries/0_stateless/03008_s3_plain_rewritable.sql

```diff
@@ -0,0 +1,22 @@
+-- Tags: no-fasttest
+-- Tag: no-fasttest -- requires S3
+
+drop table if exists test_mt;
+create table test_mt (a Int32, b Int64) engine = MergeTree() order by a
+settings disk = disk(
```

Comment:
Could you please point me to where it is located?

Reply:
_log system tables defined for CI are in `ClickHouse/tests/config/config.d/`, for example, https://github.com/ClickHouse/ClickHouse/blob/master/tests/config/config.d/blob_storage_log.xml. It is possible to define `engine` parameter in this config, where we can also add `storage_policy`, example: https://github.com/ClickHouse/ClickHouse/blob/5139ef7b9109a9d4a3822c88f77fca457ef41887/tests/integration/test_system_logs/configs/config.d/system_logs_engine.xml#L3
We could add `s3_plain_rewritable` for `blob_storage_log` from the example above. Possibly it will be better to do this only for a tests run which runs all tests on top of s3, this can be checked here https://github.com/ClickHouse/ClickHouse/blob/5139ef7b9109a9d4a3822c88f77fca457ef41887/tests/config/install.sh#L159. E.g. in this `if` branch we could substitute blob_storage_log.xml to a different one (with a different storage policy) instead of what is added here https://github.com/ClickHouse/ClickHouse/blob/5139ef7b9109a9d4a3822c88f77fca457ef41887/tests/config/install.sh#L22